src/main/resources/static/scripts.js

```diff
@@ -7,4 +7,9 @@ function lightModeToggle() {
     } else {
         button.innerHTML = "Light Mode";
     }
+}
+
+function keybindPopup() {
+    const element = document.getElementById("keybind-popup");
+    element.classList.toggle("hidden");
```

Comment:
This script will lead to the eventual pop-up window for the keybinds. Everything else for it should be CSS/HTML-based.